src/chart/tree/TreeSeries.ts

```diff
@@ -112,6 +113,16 @@ export interface TreeSeriesOption extends
     data?: TreeSeriesNodeItemOption[]
 }
 
+interface TreePathInfo {
+    name: string
+    dataIndex: number
+    value: number
+}
+
+interface TreeSeriesCallbackDataParams extends CallbackDataParams {
+    treePathInfo?: TreePathInfo[]
```

Comment:
Ignore the fact that the internal method has been named `wrapTreePathInfo`, is it better to call this property `treeAncestors` when providing to developers? `path` can mean from an arbitrary node to another arbitrary node.

Reply:
Agreed. Thanks for the suggestion.